tests/helpers.py

```diff
@@ -62,35 +65,47 @@ def gdb_run_cmd(cmd: str, before: List[str]=[], after: List[str]=[], target: str
     return result
 
 
-def gdb_run_silent_cmd(cmd, before: List[str]=[], after: List[str]=[], target: str=PATH_TO_DEFAULT_BINARY, strip_ansi: str=STRIP_ANSI_DEFAULT) -> str:
+def gdb_run_silent_cmd(cmd, before: Iterable[str]=(), after: Iterable[str]=(),
+                       target: str=PATH_TO_DEFAULT_BINARY,
+                       strip_ansi: bool=STRIP_ANSI_DEFAULT) -> str:
     """Disable the output and run entirely the `target` binary."""
-    before += ["gef config context.clear_screen False",
-               "gef config context.layout '-code -stack'",
-               "run"]
+    before = [*before, "gef config context.clear_screen False",
```

Comment:
This had a bug!

Reply:
but doesn't the bug still exist like this?

Reply:
oh, no, i had a misconception. it's ok :)